snapcraft.yaml

```diff
@@ -41,14 +41,14 @@ parts:
     source-type: git
     source-tag: '42.0'
     plugin: meson
-    meson-parameters: [--prefix=/usr]
-    organize:
-      snap/gnome-sudoku/current/usr: usr
+    parse-info: [usr/share/metainfo/org.gnome.Sudoku.appdata.xml]
+    meson-parameters:
+      - --prefix=/usr
     build-packages:
       - libqqwing-dev
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+      craftctl default
+      craftctl set version=$(git describe --tags --abbrev=10)
```

Comment:
Since the [release version is in the upstream appdata file](https://gitlab.gnome.org/GNOME/gnome-sudoku/-/blob/master/data/org.gnome.Sudoku.appdata.xml.in#L56), I think we can remove this line.

Reply:
Done.